server.js

```diff
@@ -1,32 +1,93 @@
-import express from 'express'
-import bodyParser from 'body-parser'
+/* eslint-disable linebreak-style */
+/* eslint-disable */
+
+import express, { response } from 'express'
 import cors from 'cors'
+import listEndpoints from 'express-list-endpoints'
+
+import netflixData from './data/netflix-titles.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+// middlewares to enable cors and json parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// endpoint to get list of endpoints
+app.get('/', (request, response) => {
+  response.send(listEndpoints(app))
+})
+
+// endpoint to get all movies
+app.get('/movies', (request, response) => {
+  response.json(netflixData)
+})
+
+// query parameters, filtering data
+app.get('/movies', (request, response) => {
```

Comment:
it is the same endpoint so if it will find the first version will execute it and send the response, so filtering will be not accesable.